### ISSUE TITLE
Include clstm

### ISSUE THREAD

**Issue by kba** [kba/kraken-docker#5]

**Comment by crytoy** [comment]
Have you integrated CLSTM?
How can I run ( train.sh )?

**Comment by kba** [comment]
No, I haven't, but if you want to run clstm in docker, there's [this PR](https://github.com/tmbdev/clstm/pull/121) and a [container on dockerhub of xlight's clstm fork](https://github.com/xlight/clstm)

**Comment by crytoy** [comment]
I have downloaded [your docker of kraken](https://hub.docker.com/r/kbai/kraken/), and also downloaded [xligh docker of clstm](https://hub.docker.com/r/xlight/clstm/), but now how do I use Kraken's train.sh or train using clstm?

**Comment by kba** [comment]
If by train.sh you mean the [shell script by @mittagessen](https://raw.githubusercontent.com/mittagessen/kraken-vagrant/master/training/train.sh), it contains this:

```
#!/bin/bash
set -x
set -a
sort -R manifest.txt > /tmp/manifest2.txt
sed 1,100d /tmp/manifest2.txt > train.txt
sed 100q /tmp/manifest2.txt > test.txt

report_every=1000
save_every=1000
maxtrain=50000
target_height=48
dewarp=center
display_every=1000
test_every=1000
hidden=100
lrate=1e-4
save_name=arabic
clstmocrtrain train.txt test.txt
```

You have to convert this into a docker call, see the [README.md](https://github.com/kba/clstm/blob/44016cdf602921ce6a90a9f8f3fb040f710b8ecb/README.md#installation-using-docker) in https://github.com/tmbdev/clstm/pull/121 for the basic idea. (Note to self: Should merge that one one of these days)
